lib/mia-js.js

```diff
@@ -125,13 +125,13 @@ function MiaJs() {
      * @returns {*}
      * @private
      */
-    var _initDataFunctions = function () {
-        var projectInitFunction = Shared.init();
-        for (var index in projectInitFunction) {
-            if (projectInitFunction[index]) {
-                if (projectInitFunction[index]["init"] && _.isFunction(projectInitFunction[index]["init"])) {
-                    projectInitFunction[index]["init"]();
-                }
+    const _initDataFunctions = async () => {
+        const projectInitModules = Shared.init();
+        for (const key in projectInitModules) {
+          if (!projectInitModules.hasOwnProperty(key)) continue;
+          const initModule = projectInitModules[key];
+            if (initModule["init"] && _.isFunction(initModule["init"])) {
+                await initModule["init"]();
```

Comment:
This would be faster if it is done with map and promise.all instead of for loop and await on each item.
Of course if the functionality of this chain are not dependent on each other and they could be done in parallel